src/map/entities/charentity.cpp

```diff
@@ -1020,7 +1020,7 @@ void CCharEntity::OnAbility(CAbilityState& state, action_t& action)
         action.id = this->id;
         action.actiontype = PAbility->getActionType();
         //#TODO: unoffset this
```

Comment:
Can remove these `//#TODO: unoffset this` messages since the TODO has finally be fulfilled! 😉

Reply:
removed